searxng/searxng
# Update onion engines to v3

## What does this PR do?

Ahmia's new URL from https://ahmia.fi

I also removed not_evil which has been down for a while now: https://old.reddit.com/r/onions/search/?q=not+evil&restrict_sr=on&t=year

## Why is this change important?

Tor is currently deprecating v2 onion addresses https://blog.torproject.org/v2-deprecation-timeline

## How to test this PR locally?

1. Run Tor with default settings.
2. Add or uncomment these lines in your `settings.yml`:
```
proxies:
    all://:
        - socks5h://127.0.0.1:9050
    using_tor_proxy: True
    extra_proxy_timeout: 5.0
```
3. Run searxng.
4. Try some query in the onions category, such as `!onions dogecoin`.

Alternatively, you can try it in http://w5rl6wsd7mzj4bdkbuqvzidet5osdsm5jhg2f7nvfidakfq5exda5wid.onion

Expected behaviour:
1. Some garbage results.
